docusaurus/docs/user-docs/intro.md

```diff
@@ -9,115 +9,46 @@ sidebar_label: Welcome!
-| --------------------- | ------------------------------------------------------------------------------------------------------- |
-| First name            | Write your first name in the textbox.                                                                   |
-| Last name             | Write your last name in the textbox.                                                                    |
-| Email                 | Write your complete email address in the textbox.                                                       |
-| Username              | (optional) Write a username in the textbox.                                                             |
-| Interface language    | Among the drop-down list, choose a language for your Strapi application interface.                      |
-| Interface mode        | Among the drop-down list, choose a mode for your Strapi application interface: either "Light mode" or "Dark mode". Note that by default, the chosen mode for a Strapi application is based on the browser's mode. |
+## What you will find here
 
-4. Click on the **Save** button.
+The table of content in this User Guide displays 7 main sections.
 
-### Changing your password
+Clicking on any of the following cards will direct you to the introductory page for that section, with additional details and concepts:
 
-To change the password of your account:
-
-1. Go to your administrator profile.
-2. Fill in the password-related options:
-
-| Password modification |                                                                                                         |
-| --------------------- | ------------------------------------------------------------------------------------------------------- |
-| Current password      | Write your current password in the textbox. <br/> 💡 You can click on the eye icon for the password to be shown. |
-| Password              | Write the new password in the textbox. <br/> 💡 You can click on the eye icon for the password to be shown.      |
-| Password confirmation | Write the same new password in the textbox. <br/> 💡 You can click on the eye icon for the password to be shown. |
-
-3. Click on the **Save** button.
-
-
----
-
-Congratulations on being a new Strapi user! You're now ready to discover all the features and options that Strapi has to offer!
+<CustomDocCardsWrapper>
+  <CustomDocCard emoji="📝" title="Content Manager" description="Manage and publish all content types created with the Content-type Builder." link="/user-docs/content-manager" />
+  <CustomDocCard emoji="📚" title="Content Type Builder" description="Create, manage and publish more than one single entries." link="/user-docs/content-type-builder" />
+  <CustomDocCard emoji="🗃️" title="Media Library" description="Upload and manage all assets and organize them into folders." link="/user-docs/media-library" />
+  <CustomDocCard emoji="📅" title="Releases" description="Arrange entries into containers for publishing and unpublishing actions." />
```

Comment:
I believe there's an issue with the Releases card, the content is written in purple contrary to all other cards descriptions. Might be because the link is missing?

<img width="1097" alt="Screenshot 2024-02-28 at 09 58 55" src="https://github.com/strapi/documentation/assets/19183360/469e7a03-fa28-4b55-b79c-ec14e90011bd">

Reply:
Yes it was, I've included it now